remotespark/wrapperkernel/sparkkernelbase.py

```diff
@@ -224,4 +238,4 @@ def _repeat_fatal_error(self):
         error = conf.fatal_error_suggestion().format(self._fatal_error)
         self._logger.error(error)
         self._ipython_display.send_error(error)
-        raise ValueError(self._fatal_error)
+        return self._complete_cell()
```

Comment:
`_abort_with_fatal_error` returns `None` now because it has no `return` statement.